DXMainClient/Domain/Multiplayer/PlayerInfo.cs

```diff
@@ -22,6 +23,7 @@ public PlayerInfo(string name, int sideId, int startingLocation, int colorId, in
             StartingLocation = startingLocation;
             ColorId = colorId;
             TeamId = teamId;
+            Ping = -1;
```

Comment:
Should set this as the default in the property instead. No need to add lines to the constructors then.